__tests__/lib/index.test.js

```diff
@@ -110,14 +112,9 @@ describe( 'go-search-replace', () => {
 			expect( outFile ).not.toContain( 'thatdomain.com' );
 		} );
 
-		it( 'throws a new Error when the script/binary returns a non-zero exit code', async () => {
-			try {
-				// await replace( readableStream, [ 'thisdomain.com', 'thatdomain.com' ], nonZeroExitCodeScript );
-				await testHarness( [ 'thisdomain.com', 'thatdomain.com' ], nonZeroExitCodeScript );
-			} catch ( err ) {
-				expect( err ).toEqual( 'The search and replace process exited with a non-zero exit code: 1' );
-			}
-		} );
+		it( 'throws a new Error when the script/binary returns a non-zero exit code', () =>
+			expect( testHarness( [ 'thisdomain.com', 'thatdomain.com' ], nonZeroExitCodeScript ) ).rejects.toEqual( 'The search and replace process exited with a non-zero exit code: 1' ),
+		);
```

Comment:
Our ESLint configuration does not like conditional `expect`. Luckily, we can replace the `try...catch` block with the [`.rejects` matcher](https://jestjs.io/docs/asynchronous#resolves--rejects).